### ISSUE TITLE
Unable to use G-Keys of G930 Headset to Play/Pause, or switch track

### ISSUE THREAD

**Issue by Saucy** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#1166]
- No other issue.
- Checked the FAQ (I have no Google Play Music extension installed)
- I can reproduce the issue.

**OS:**
Windows 7 Ultimate 64bit

**Issue Descriptions:**
Unable to use/setup `Play/Pause`, `Next Track`, `Previous Track` using the G-Keys of my [Logitech G930 wireless headset](http://gaming.logitech.com/en-us/product/g930-7-1-wireless-gaming-headset). G-Keys works in any other media program (Spotify, foobar2000, etc).

**Steps to Reproduce:**
Desktop Settings -> Hotkeys. Unable to add G-Keys to the `Play/Pause`, `Next Track`, `Previous Track`-hotkeys.

Let me know if there's anything else you'd need to know!

**Comment by MarshallOfSound** [comment]
Can you try figure out what keycode event the headset is firing.

Run this executable: 
[LLKeyboardTest.zip](https://github.com/MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-/files/304683/LLKeyboardTest.zip)

While it is running, press your headset buttons and tell me what numbers appear in the console window

**Comment by Saucy** [comment]
Play/Pause: 179
Next: 176
Previous: 177

**Comment by MarshallOfSound** [comment]
Ummmmmmmm

Well this is awkward.  Those are exactly the numbers we are looking for...

https://github.com/MarshallOfSound/node-lowlevel-keyboard-hook-win/blob/master/ll_keyboard_hooks.js#L32

Are you sure the keys don't "just work".  Have you actually tried them 😆

**Comment by MarshallOfSound** [comment]
I.e  Play some music and try and pause it with your headset

**Comment by Saucy** [comment]
I found out where the problem was, it was my G930 drivers!
I'm gonna write how "I solved it" just in case someone else with Logitech headset has the same issue:

When writing this issue I used the latest Logitech Gaming Software (v8.83.85). After searching around for similar problems with media programs and the G-Keys, I found [this video on headset disconnecting very often](https://www.youtube.com/watch?v=nA-kvcvZ40M) (unrelated to my media buttons-problem, but I've had disconnect problems with my headset since I bought it).

Anyway, I tested the version he recommended (v8.58.183 - can be [downloaded here](http://support.logitech.com/en_us/software/lgs)), now my media buttons works!

Thanks for your help @MarshallOfSound. Never thought it would end up being a (latest) driver issue!
